addons/stock/models/stock_move.py

```diff
@@ -703,6 +703,7 @@ def _assign_picking(self):
             else:
                 recompute = True
                 picking = Picking.create(move._get_new_picking_values())
+                move._picking_post_process(picking)
```

Comment:
I would:
- move this below `move.write({'picking_id': picking.id})` (no need to pass picking as argument)
- pass an argument `new` to know whether or not the picking was just created